configurator/src/main/java/com/devonfw/tools/ide/migrator/Migrations.java

```diff
@@ -147,6 +147,36 @@ public static MigrationImpl devon4j() {
         .pom().replaceProperty("devon4j.version", "3.1.1") //
         .replaceProperty("jackson.version", "2.9.9.20190727") //
         .and() //
+        .next().to(VersionIdentifier.ofDevon4j("3.2.0")) //
+        .pom().replaceProperty("devon4j.version", "3.2.0") //
+        .replaceString("com.devonfw.module.test.common.api.category.CategorySystemTest", "system")
+        .replaceString(
+            "com.devonfw.module.test.common.api.category.CategoryComponentTest,com.devonfw.module.test.common.api.category.CategorySubsystemTest,com.devonfw.module.test.common.api.category.CategorySystemTest",
+            "component,subsystem,system")
+        .replaceString(
+            "com.devonfw.module.test.common.api.category.CategorySubsystemTest,com.devonfw.module.test.common.api.category.CategorySystemTest",
+            "subsystem,system")
+        .replaceDependency(new VersionIdentifier("junit", "junit", null),
+            new VersionIdentifier("org.junit.jupiter", "junit-jupiter-engine", null) {
+              {
+                setScope("test");
+              }
```

Comment:
nice-to-have-improvment:
Could you add a constructor with scope to avoid this ugly nested annonymous classes?
Can you add a constant `SCOPE_TEST` to `VersionIdentifier`?

Reply:
done